src/lib/Scenes/Artwork/Artwork.tsx

```diff
@@ -1,47 +1,126 @@
 import React from "react"
-import { Dimensions, ScrollView } from "react-native"
+import { Dimensions, FlatList } from "react-native"
 import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
 import { AboutArtistFragmentContainer as AboutArtist } from "./Components/AboutArtist"
-import { AboutWorkFragmentContainer as AboutWork } from "./Components/AboutWork"
-import { ArtworkActionsFragmentContainer as ArtworkActions } from "./Components/ArtworkActions"
-import { ArtworkAvailabilityFragmentContainer as ArtworkAvailability } from "./Components/ArtworkAvailability"
+import { AboutWork } from "./Components/AboutWork"
+import { ArtworkAvailability } from "./Components/ArtworkAvailability"
 import { ArtworkDetailsFragmentContainer as ArtworkDetails } from "./Components/ArtworkDetails"
+import { ArtworkHeader } from "./Components/ArtworkHeader"
 import { ArtworkTombstoneFragmentContainer as ArtworkTombstone } from "./Components/ArtworkTombstone"
-import { ImageCarouselFragmentContainer as ImageCarousel } from "./Components/ImageCarousel/ImageCarousel"
 import { OtherWorksFragmentContainer as OtherWorks } from "./Components/OtherWorks"
-import { SellerInfoFragmentContainer as SellerInfo } from "./Components/SellerInfo"
+import { SellerInfo } from "./Components/SellerInfo"
 
 interface Props {
   artwork: Artwork_artwork
 }
 
+interface State {
+  sections: Array<{
+    type: "tombstone" | "availability" | "sellerInfo" | "aboutWork" | "details" | "aboutArtist" | "otherWorks"
+    data: any
+  }>
+}
+
 export class Artwork extends React.Component<Props> {
-  render() {
+  state: State = {
+    sections: [],
+  }
+
+  componentDidMount() {
+    const { artwork } = this.props
+    const sections = []
+
+    sections.push({
+      type: "header",
+    })
+
+    sections.push({
+      type: "tombstone",
+    })
+
+    if (artwork.availability) {
+      sections.push({
+        type: "availability",
+        data: artwork.availability,
+      })
+    }
+
+    if (artwork.partner && artwork.partner.name) {
+      sections.push({
+        type: "sellerInfo",
+        data: artwork.partner.name,
+      })
+    }
+
+    if (artwork.description || artwork.additional_information) {
+      sections.push({
+        type: "aboutWork",
+        data: { description: artwork.description, additionalInformation: artwork.additional_information },
+      })
+    }
+
+    sections.push({
+      type: "details",
+    })
+
+    sections.push({
+      type: "aboutArtist",
+    })
+
+    sections.push({
+      type: "otherWorks",
+    })
+
+    this.setState({ sections })
+  }
+
+  renderItem = ({ item: { data, type } }) => {
     const { artwork } = this.props
+    switch (type) {
+      case "header":
+        return <ArtworkHeader artwork={artwork} />
+      case "tombstone":
+        return <ArtworkTombstone artwork={artwork} />
+      case "availability":
+        return <ArtworkAvailability availability={data} />
```

Comment:
This component in particular is small but was explicitly built this way because we will be adding more logic/data to it as we expand the artwork page to handle commercial works. It'll make those additions much easier if we leave it as a fragment container!